scripts/screeps-runtime-monitor.py

```
@@ -4050,6 +4208,90 @@ def runtime_summary_room_with_cpu_fields(base: dict[str, Any], cpu_room: dict[st
     return result
 
 
+def runtime_summary_capture_history_entry(room: dict[str, Any]) -> dict[str, Any]:
```

Ruff (0.15.15)

[warning] Too many branches (14 > 12)

(PLR0912)